README.md

```diff
@@ -28,31 +26,54 @@ julia> ] add Javis
 
 That's all there is to it! 😃
 
-## Testing
-
-1. Clone the repo to your computer and go into that directory:
-
-`cd Javis`
+### Dependencies
```

Comment:
Could we change this label to optional dependencies? Right now, you don't _need_ this node dependency to run Javis but just to use the `latex` function.